cmd/snap/cmd_add_skill.go

```diff
@@ -0,0 +1,65 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"github.com/ubuntu-core/snappy/client"
+	"github.com/ubuntu-core/snappy/i18n"
+)
+
+type cmdAddSkill struct {
+	Positionals struct {
+		Snap string `positional-arg-name:"snap" description:"name of the snap offering the skill"`
+		Name string `positional-arg-name:"name" description:"skill name within the snap"`
```

Comment:
I would call this one  "skill-name",  my problem otherwise is that otherwise when I read the help:

snap name type

I read that as one thing confusingly